### DISCUSSION TITLE
Student Pack Recused

### DISCUSSION THREAD

**Discussion by apriolo** [community/community#144303] [GitHub Education]
### Hi everyone,

Hello, my name is Arian, and I'm from Brazil, I'm currently studying a bachelor's degree in information systems, at a university here called, Centro Universitário de Bauru, I signed up for the student package, and my answer was denied because I need my email- school mail on github is confirmed, but my university does not provide email, only a 365 package to use word, excel etc, I have attached my student card and my enrollment document, how should I proceed to get my pack?
, but email i cant confirm, how can i get a student pack now?

**Comment by huddmax** [comment]
Oi, Arian!

Eu também tive dificuldades com a verificação do GitHub Education, então aqui vão algumas dicas que podem te ajudar:

1. **Documentação**: Certifique-se de que seu **cartão de estudante e documento de matrícula** mostrem claramente seu nome completo, o nome da instituição e o período em que está matriculado.

2. **Explicação Detalhada**: No campo de justificativa, explique que sua universidade não oferece e-mail institucional, mas que você possui a comprovação de matrícula e o acesso ao pacote Office 365.

3. **Contato com o Suporte do GitHub Education**: Caso ainda enfrente problemas, entre em contato diretamente com o suporte do GitHub Education e explique a situação. Às vezes, uma revisão manual pode ajudar nesses casos em que a universidade não fornece e-mail institucional.

Espero que essas dicas ajudem e que consiga seu Student Pack! Boa sorte!

**Comment by huddmax** [comment]
Esse acesso ao Office 365 é em qual email? Porque na maioria das vezes a faculdade disponibiliza o Office 365 já com o seu email institucional da faculdade
